readme.md

```diff
@@ -673,6 +676,8 @@ You can use any assertion library instead of or in addition to the built-in one,
 
 This won't give you as nice an experience as you'd get with the [built-in assertions](#assertions) though, and you won't be able to use the [assertion planning](#assertion-planning) ([see #25](https://github.com/avajs/ava/issues/25)).
 
+You'll have to configure AVA to not fail tests if no assertions are executed, because AVA can't tell if custom assertions pass. Set the `failWithoutAssertions` option to `false` in AVA's [`package.json` configuration](#configuration).
```

Comment:
> This feature, while useful, is going to hit a lot of people. Any chance we could add support for assertions throwing AssertionError?

That's not the problem, actually. Any custom assertion that throws an error causes the test to fail, just because tests shouldn't throw errors.

The issue is when all custom assertions pass and no errors are thrown. The intent is for the test to pass, but we don't know that custom assertions were executed. So it'll fail. That's why I added this option.

IMHO the question is what the default behavior should be.